tiledb/bioimg/__init__.py

```diff
@@ -0,0 +1,10 @@
+from typing import Any, Optional
+
+import tiledb
+
+ATTR_NAME = "intensity"
+
+
+def open(uri: str, *args: Optional[Any], **kwargs: Optional[Any]) -> tiledb.Array:
```

Comment:
1. This is an internal helper function, probably shouldn't live in the main public entry point of the package. I'd put it in `base.py` but this creates a cyclic import with `openslide.py` so perhaps a new `utils.py`? As a bonus, a new module could be used to move there all the private functions of `base.py` and leave just the base classes.
2. The original `tiledb.open` signature doesn't have to be preserved, it can be simplified to `(uri, mode="r")`.

Reply:
1. I moved `open` and some funcs from `base` module to a newly created under `bioimg`. However one of them creates again cyclic imports so I left it under `base`. Ideally there should have been 2 utils modules under `converters` package where we would store all the mangled funcs of the base module without cyclic imports and one under `bioimg` that would host `_open` which is used both from `converters` modules but also `openslide`.

2. Is it bad if it is preserved?

Reply:
1. I moved open and some funcs from base module to a newly created helpers.py under bioimg. However one of them creates again cyclic imports so I left it under base. Ideally there should have been 2 utils modules one under converters package where we would store all the mangled funcs of the base module without cyclic imports and one under bioimg that would host _open which is used both from converters modules but also openslide.

2. Is it bad if it is preserved?